apps/ocpp/call_handlers/errors/data_transfer.py

```
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+from channels.db import database_sync_to_async
+from django.utils import timezone
+
+from ... import store
+from ...models import DataTransferMessage
+from ..types import CallErrorContext
+
+
+async def handle_data_transfer_error(
+    consumer: CallErrorContext,
```

Ruff (0.14.14)

[warning] Unused function argument: `consumer`

(ARG001)